src/pages/Lend/Filter/FilterMenuSection.vue

```diff
@@ -49,13 +51,29 @@ export default {
 
 	.filter-items-container {
 		display: none;
-		background-color: #F6FBF6;
+
+		ul {
+			list-style-type: none;
+			margin-bottom: 0;
+		}
+
+		.ais-HierarchicalMenu {
```

Comment:
Kinda odd to have these here but I suspect we can reuse them by adding other components too.

Reply:
Exactly. I chose to leave these styles in case we needed to revert to the built in `HierarchicalMenu` component. Other styles for built in components can go here too.